### ISSUE TITLE
[Feature] Support for Grub/BIOS control

### ISSUE THREAD

**Issue by P2theG1** [ah-/anne-key#59]
I would like to request that the ability to use the keyboard in a Grub menu/BIOS is added, for example being able to use the F10 (Via FN+0) to access the BIOS on boot up, initially as a feature operated via USB cable connection and then also via Bluetooth Connection at a future point in time (If Bluetooth functionality of this type is possible with this keyboard), currently I cannot use my Anne Pro as my main keyboard because I need to have use of the F1-F12 (And other) keys at boot up, thanks for the work that you are doing to expand the Anne Pro's features & making it more stable

**Comment by ah-** [comment]
Yes, that sounds useful. Probably nothing we can do about BT support, but USB should be doable.

**Comment by P2theG1** [comment]
Awesome, I didn't think Bluetooth would be possible, I just thought I'd mention it as an extra, also on top of the F10 key it'd be great if the FN layer arrow keys also worked so that the BIOS/GRUB/UEFI could be navigated (FN+W/A/S/D or FN+I/J/K/L), If theirs any way I can help with testing of the firmware and troubleshooting I'd be more than happy to do so, PG
